lp-core-platform/lp-cp-apis/src/main/java/eu/learnpad/core/impl/cw/XwikiBridge.java

```diff
@@ -95,20 +97,38 @@
 	@Inject
 	@Named("current")
 	private DocumentReferenceResolver<String> documentReferenceResolver;
-
+/*
+ * Why we have this attribute here? 
+ * A controller in the XwikiBridge is inherited by the Bridge class
+ * by means of its CoreFacade 
+ * 
+ * I suggest to remove it, and using the reference "this.corefacade" instead.
+ */
 	private XwikiController cwController;
```

Comment:
could we close this PR first and then work on the re-definition of such methods?